core/orch_test.go

```diff
@@ -785,6 +785,16 @@ func TestSufficientBalance_IsNotSufficient_ReturnsFalse(t *testing.T) {
 	assert.False(orch.SufficientBalance(manifestID))
 }
 
+func TestSufficientBalance_OffChainMode_ReturnsTrue(t *testing.T) {
+	n, _ := NewLivepeerNode(nil, "", nil)
+	manifestID := ManifestID("some manifest")
+	orch := NewOrchestrator(n)
+	assert.True(t, orch.SufficientBalance(manifestID))
+
+	orch.node = nil
```

Comment:
Can we add tests for the other cases included in the guard clause as well?

Reply:
orch.node.Recipients and orch.node.Balances should be nil in the first assert check, the wrapping node should be nil in the second assert check. You want to explicitly set a recipient with Balances nil and vice versa as well?

Reply:
Yeah I think it would be good to include to cover:

- `orch.node != nil && orch.node.Recipient = nil`
- `orch.node != nil && orch.node.Recipient != nil && orch.node.Balances = nil`

Reply:
Fair enough, added the test cases.